### PR TITLE
Added keyval_to_unicode

### PR DESCRIPTION
**Pull request by trolleyman**
Think I've got the hang of this now

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks for your PR ! You forgot the `use std::char;`. Could you also add an empty line between your declaration and your if / else please ? And lastly, could you squash your commits ? Normally I don't mind but 5 commits for 12 lines seems too much. I'll merge once it's done.

File: gdk-sys/src/lib.rs [hunk 1]
```diff
@@ -685,6 +685,7 @@ extern "C" {
     // Gdk Key Handling                                                  NOT OK
     //=========================================================================
     pub fn gdk_keyval_name                         (keyval:c_uint) -> *mut c_char;
+    pub fn gdk_keyval_to_unicode                   (keyval:c_uint) -> u32;
```
Comment (gkoz) [inline gdk-sys/src/lib.rs]: It would be consistent with the ffi style to use `c_uint` not `u32` here.

Comment (GuillaumeGomez) [inline gdk-sys/src/lib.rs]: @gkoz: No, take a look [here](https://developer.gnome.org/gdk3/stable/gdk3-Keyboard-Handling.html#gdk-keyval-to-unicode). `u32` is perfectly fine.

Comment (gkoz) [inline gdk-sys/src/lib.rs]: Yeah, okay, while `u32` is unusual I suppose it does reflect `guint32` more precisely.

Comment (GuillaumeGomez) [inline gdk-sys/src/lib.rs]: When the size of the type is specified, I prefer to use its equivalent in Rust.

**Comment by trolleyman** [comment]
Sure, one sec

**Comment by GuillaumeGomez** [comment]
@trolleyman: Take your time, we're not in a hurry. ;-)

**Comment by GuillaumeGomez** [comment]
Perfect like that ! (It misses the empty line but it's not important since the code is only 2 lines long). Can you squash your commits please ?

**Comment by trolleyman** [comment]
Sorry but I have no idea how to do that, I've just been editing the files through the browser...

**Comment by trolleyman** [comment]
I might just delete this PR and start again...?

**Comment by trolleyman** [comment]
Yeah, I'll do that, I'll be back.

**Comment by GuillaumeGomez** [comment]
Hum... Normally you just use this command :

```Shell
> git rebase -i HEAD~x
```

Where `x` is the number of commits which would have been squashed (in your case 6).

**Pull request closed without merging**